src/components/EmailList/EmailList.jsx

```diff
@@ -0,0 +1,63 @@
+import EmailItem from "../EmailItem/EmailItem";
+import SearchBar from "../SearchBar/SearchBar";
+import EmailString from "../EmailString"
+import { useEffect, useState } from "react"
+
+function EmailList() {
+
+    const [emailString, setEmailString] = useState(false)
+    const emailItems = [];
+
+    function fetchJSON() {
+        
+        useEffect(() => {
+            fetch("https://email-client-api.dev.io-academy.uk/emails")
+            .then(response => response.json())
+            .then(data => {
+                console.log(data)
+                setEmailString(data)
+                return(
+                    emailString
+                )
+            })
+        }, [])
+
+    }
+
+    fetchJSON();
+
+    const [filteredEmailItems, setFilteredEmailItems] = useState(emailString);
+
+    const handleSearch = (searchQuery) => {
+        const filteredItems = EmailString.filter(email => {
```

Comment:
Your state variable is called `emailString` - small `e`. This code will not work.